### DISCUSSION TITLE
Berty should also be open source in the process

### DISCUSSION THREAD

**Discussion by sudwhiwdh** [berty/berty#4567] [General]
Don't know yet if this is the place for my concern. And I would like to try courageously.

I have read and heard that Berty uses quite a few non-free solutions in his development and communication.
GitHub. Discord. Zenhub. Crowdin. Twitter.

I would like to refer to your own blog article:

https://berty.tech/blog/reasons-to-berty/https://berty.tech/blog/reasons-to-berty/

> 
> 8 Transparent to the core. 👓
> 
> Berty champions the Open Source state of mind, in its vision, product, process, and team.
> 

My request to you would therefore be to also focus on ethical and free solutions in the process when it comes to platforms and tools for the development of Berty or contact channels with the community.

**Comment by jefft0** [comment]
Most of the dependencies are on GitHub, including IPFS. This has not been a problem. There are no plans to change this. The Berty Discord "rules" channel has this statement:

> We are aware that Discord is not a privacy-friendly tool, but it is a great way to interact with our community about Berty itself.
> Weighing the minimal privacy risks of discussing an open-source project against the benefits of providing a familiar, accessible platform to encourage community growth, we believe it's in our interest to continue using it.

**Reply by sudwhiwdh** [reply]
Is the Berty team open to me creating GitHub issues for the platforms and tools mentioned and suggesting a switch to suitable open source alternatives?
And if so, in which repository would that make sense?

**Reply by sudwhiwdh** [reply]
And I read what you write about Discord and wonder if there aren't alternatives that offer the same qualities. I read your comment as saying that it's not about a product like Discord, but about the qualities for which it should be used, right?

**Comment by jefft0** [comment]
The tools we have now are working well and users are familiar with them. It is disruptive to make such a big change. What is the security threat or risk to the project that requires such changes?

**Comment by sudwhiwdh** [comment]
What kind of Amazon infrastructure do you use for Berty and why? https://github.com/berty/berty/releases/tag/v2.468.0
Or is aws not = Amazon?

**Comment by sudwhiwdh** [comment]
Your Twitter account https://twitter.com/berty is no longer a public information channel. Please switch to the Fediverse!

**Reply by LFGaming** [reply]
Hi @sudwhiwdh, why do you say that the Berty twitter is longer a public information channel? And how would Fediverse improve on this issue?

**Reply by sudwhiwdh** [reply]
Feel free to try it out.
Go to https://twitter.com/berty and try to view the page. If you have a Twitter account and/or are logged in, log out first and then visit https://twitter.com/berty again.
The Fediverse is an alternative in the sense that it offers people a federated approach to sharing news and information rather than a centralized approach. And the posts - e.g. on a Mastodon instance - remain visible even without a centralized account. With Twitter, I now have to log in. And I don't want an account there.

**Reply by LFGaming** [reply]
Thanks, I completely forgot about the fact that you now have to log in to see the latest tweets too.

**Comment by ghost** [comment]
Just to add another perspective to this discussion: The optics of an open-source, non-profit, community-led, and privacy-focused app using Twitter and Discord are not the best. This also has the potential to conflict with the goal of sustainable fundraising for this critical work.

Who is the team reaching on Twitter/Discord who is deeply interested in noncommercial community projects, yet doesn't use any decentralized alternatives to those tools?
